libsqlite3-sys/src/wasm32_unknown_unknown.rs

```diff
@@ -0,0 +1,181 @@
+#[cfg(not(feature = "bundled"))]
+compile_error!("wasm32-unknown-unknown must be built with '--features bundled'");
+
+use crate::{sqlite3_file, sqlite3_vfs, sqlite3_vfs_register, SQLITE_IOERR, SQLITE_OK};
+use std::os::raw::{c_char, c_int, c_void};
+use std::ptr::null_mut;
+
+#[no_mangle]
+pub unsafe extern "C" fn sqlite3_os_init() -> c_int {
+    let vfs = sqlite3_vfs {
+        iVersion: 1,
+        szOsFile: 0,
+        mxPathname: 1024,
+        pNext: null_mut(),
+        zName: "libsqlite3-sys".as_ptr() as *const c_char,
```

Comment:
```suggestion
        zName: "libsqlite3-sys\0".as_ptr() as *const c_char,
```

`zName` is a c string.